crates/biome_js_analyze/src/analyzers/nursery/use_shorthand_function_type.rs

```diff
@@ -95,6 +95,10 @@ impl Rule for UseShorthandFunctionType {
 
     fn run(ctx: &RuleContext<Self>) -> Self::Signals {
         let query = ctx.query();
+        // If there are comments, it's not a single call signature.
+        if query.syntax().has_comments_direct() {
+            return None;
+        }
```

Comment:
I think we should still report cases with inner comments, however, we should not suggest any fixes. Thus, this check should be moved in `action`.

Reply:
thanks for comment.
fix 797c48b